CONTRIBUTING.md

```diff
@@ -29,6 +29,25 @@ This is a rough outline of what a contributor's workflow looks like:
 
 Thanks for contributing!
 
+### Local setup
```

Comment:
There's local.md document that describes local setup. Is this section somehow different?

Reply:
it's the same but the reflection from the contributor standpoint. Actually to address the same, maybe in a more concise and focused way. 

Basically this an attempt to differentiate generic playground vs contributor testing setup as in #446 problem statement

Reply:
From what I see in the changes, `deploy-full-local-setup` target is still being used in both cases, playground and contributor.
It is mentioned in README.md's  QuickStart section. Is this PR considered as just an initial phase of #446 implementation?

Reply:
It's kind of layered. First, you deploy stable with `deploy-full-local-setup` then apply test candidate with `upgrade-candidate`.  In QuickStart it is `deploy-full-local-setup`  only

Reply:
OK I see, makes sense